File: beecrowd/1/1015/1015.cs
using System;

class URI {

    static void Main(string[] args) {
        double[] x = new double[2];
        double[] y = new double[2];
        for (int i = 0; i < 2; i++) {
            string[] input = Console.ReadLine().Split(' ');
            x[i] = double.Parse(input[0]);
            y[i] = double.Parse(input[1]);
        }
        Console.WriteLine("{0}", (Math.Sqrt( Math.Pow(x[1] - x[0], 2) + Math.Pow(y[1] - y[0], 2) )).ToString("0.0000"));
    }
}

File: beecrowd/1/1015/1015.py
import math
x = []
y = []

for i in range(2):
    line = input().split()
    x.append(float(line[0]))
    y.append(float(line[1]))


print("%.4f" % (math.sqrt(((x[1] - x[0]) ** 2) + ((y[1] - y[0]) ** 2))))
